Index: prototype-adapter.src.js
===================================================================
--- prototype-adapter.src.js	(revision 9031)
+++ prototype-adapter.src.js	(working copy)
@@ -146,6 +146,7 @@
 		options = options || {};
 		options.delay = 0;
 		options.duration = (options.duration || 500) / 1000;
+		options.afterFinish = options.complete;
 
 		// animate wrappers and DOM elements
 		if (hasEffect) {
@@ -158,6 +159,9 @@
 			for (key in params) {
 				el.attr(key, params[key]);
 			}
+			if( options.complete ) {
+				options.complete();
+			}
 		}
 
 		if (!el.attr) { // HTML element, #409
