Index: tiny_mce_src.js
===================================================================
--- tiny_mce_src.js	(revision 9732)
+++ tiny_mce_src.js	(working copy)
@@ -13515,7 +13515,7 @@
 		},
 
 		hide : function() {
-			var self = this, doc = t.getDoc();
+			var self = this, doc = self.getDoc();
 
 			// Fixed bug where IE has a blinking cursor left from the editor
 			if (isIE && doc)
